apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -374,8 +406,34 @@ const TutorialExplorer = React.createClass({
      */
     findMatchingTag(filterGroup, tutorialTags) {
       return filterGroup.some(filterName => tutorialTags.split(',').indexOf(filterName) !== -1);
-    }
+    },
 
+    /* Returns an array of unique organization names from the set of tutorials,
+     * sorted alphabetically, truncated and ellipsed.
+     *
+     * @param {Array} tutorials - Array of tutorials.
+     * @return {Array} - Array of strings.
+     */
+    getUniqueOrgNamesFromTutorials(tutorials) {
+      // Filter out tutorials with do-not-show tag.
+      const availableTutorials = tutorials.filter(t => {
+        return t.tags.split(',').indexOf("do-not-show") === -1 && t.orgname !== "do-not-show";
+      });
+
+      // Construct array of unique org names from the tutorials.
+      let uniqueOrgNames = [...new Set(availableTutorials.map(t => t.orgname))];
```

Comment:
Is the goal of using `Set` just to get unique? If so, you could instead use `_.uniq`

Reply:
Yes.  I thought it cool to use ES6 all the way but if we still like _ mixed in, then I can do that.